tests/unittests/DeviceManagerTest.cpp

```diff
@@ -25,13 +27,15 @@
 using namespace glow;
 using namespace glow::runtime;
 
+class DeviceManagerTest : public ::testing::TestWithParam<BackendKind> {};
```

Comment:
can you store backendkind in this class? this way we'll avoid having `GetParams()` in other places. And `GetParams()` does not really tell what is extracted.

Reply:
Sorry I don't understand what problem you're trying to solve here.

Reply:
mostly readability, e.g,
```compileFunctions(GetParam(), module.get(), backing);```
and in other places where you call `GetParam()`. It's not clear what GetParam() refers to.